artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/BridgeImpl.java

```diff
@@ -927,9 +927,7 @@ protected void connect() {
                scheduleRetryConnectFixedTimeout(this.retryInterval);
                return;
             } else {
-               if (logger.isDebugEnabled()) {
-                  logger.debug("Bridge " + this + " is unable to connect to destination. Retrying", e);
-               }
+               logger.warn("Bridge " + this + " is unable to connect to destination. Retrying...");
```

Comment:
Can we use the logger class's for this, as its no longer a debug message, should have proper code etc. etc.

Reply:
@michaelandrepearce I have used the same message as in another if-else condition. is it ok?

Reply:
Not really it should have its own, it’s a different issue, and so should have its own dedicated log error code and log output.